styles.css

```diff
@@ -64,3 +64,15 @@ h2 {
     align-content: space-around;
 }
 
+.hamster {
+    z-index: -5;
+    position: absolute;
+}
+
+.sloth {
+    position: absolute;
```

Comment:
why position absolute? no `top` `left` `bottom` `right` properties set - this does something weird but it's not correct use of `position: absolute`.